tests/node/rpc/enums/AppCapabilityTypeTests.js

```diff
@@ -0,0 +1,28 @@
+const SDL = require('./../../../../dist/js/SDL.min.js');
+
+const AppCapabilityType = SDL.rpc.enums.AppCapabilityType;
+const Validator = require('./../../../Validator.js');
+
+describe('AppCapabilityTypeTests', function () {
+    it('testValidEnums', function (done) {
+        const example = 'VIDEO_STREAMING';
+        const enumVideoStreaming = AppCapabilityType.valueForKey(example);
+
+        Validator.assertNotNull(enumVideoStreaming, 'VIDEO_STREAMING returned null');
```

Comment:
@vladmu Do you need to use `Validator.assertNotNullUndefined(...)` here on line 11 instead of `Validator.assertNotNull(...)`?  `AmbientLightStatusTests.js` is using that in its `testValidEnums` function.

Reply:
Done in https://github.com/smartdevicelink/sdl_javascript_suite/pull/302/commits/4b6e18664545fe2547f9019350240bf1b713ce5b